SPECS/inih/inih.spec

```diff
@@ -0,0 +1,135 @@
+Summary:  A Simple INI file parser library
+Name:     inih
+Version:  57
+Release:  1%{?dist}
+License:  BSD
+URL:      https://github.com/benhoyt/inih
+#Source0:  https://github.com/benhoyt/inih/archive/refs/tags/r57.tar.gz
+Source0:  inih-r57.tar.gz
+Vendor:   Microsoft Corporation
+Distribution:   Mariner
+BuildRequires: gcc
+BuildRequires: gcc-c++
+BuildRequires: meson
+
+
+%description
+The inih package provides simple INI file parser which is only a couple of
+pages of code, and it was designed to be small and simple, so it's good for
+embedded systems.
+
+
+%package devel
+Summary:  Development package for %{name}
+Requires: %{name}%{?_isa} = %{version}-%{release}
+
+
+%description devel
+This package contains development files for %{name}.
+
+The inih package provides simple INI file parser which is only a couple of
+pages of code, and it was designed to be small and simple, so it's good for
+embedded systems.
+
+
+%prep
+%autosetup -n %{name}-r%{version}
+
+
+%build
+%meson
+%meson_build
+
+
+%install
+%meson_install
+
+
+%ldconfig_scriptlets
+
+
+%files
+%license LICENSE.txt
+%doc README.md
+%{_libdir}/lib%{name}.so.0
+%{_libdir}/libINIReader.so.0
+
+
+%files devel
+%{_includedir}/ini.h
+%{_includedir}/INIReader.h
+%{_libdir}/pkgconfig/inih.pc
+%{_libdir}/pkgconfig/INIReader.pc
+%{_libdir}/lib%{name}.so
+%{_libdir}/libINIReader.so
+
+
+%changelog
+* Fri Jul 28 2023 Andy Zaugg <azaugg@linkedin.com> - 57-1
+- Updated version to 57
+- Initial CBL-Mariner import from Fedora (license: BSD)
```

Comment:
I think this comment here corresponds to the license of the `spec` file which IMO is `MIT`.

The comment should probably read something like:

`Initial CBL-Mariner import from Fedora 36 (license: MIT)`
and substitute the appropriate version of Fedora where we imported it from.

Reply:
Ah, thank you. Updated the Fedora version